coops-ui/src/components/StandaloneDirectorChange/CODDate.vue

```diff
@@ -108,12 +109,19 @@ export default class CODDate extends Mixins(DateMixin) {
      * Determine the latest of the following dates:
      * - the last COD filing in filing history (from legal DB)
      * - the last AR filing in filing history (from the Legal DB)
-     * - the last pre-load Cobrs filing
+     *
+     * If the entity has no filing history, the founding date will be used.
      */
-    const lastARFilingDate = this.lastAnnualReportDate == null ? 0 : +this.lastAnnualReportDate.split('-').join('')
-    const lastCODFilingDate = this.lastCODFilingDate == null ? 0 : +this.lastCODFilingDate.split('-').join('')
-    const minCODDate = Math.max(lastARFilingDate, lastCODFilingDate)
-    return this.numToUsableString(minCODDate)
+    let minDate = null
```

Comment:
I'm not sure you need to explicitly define `minDate` as `null`. If you declare it, it will be inherently `undefined`, but i'm not sure that be an issue with how it's used in this context?